Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,78 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * moveAction event handler, launch the universal discovery widget
+         * to choose a location to move the content
+         *
+         * @method _selectLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _selectLocation: function (e) {
+            this.fire('contentDiscover', {
+                config: {
+                    title: "Select the location you want to move your content into",
+                    contentDiscoveredHandler: Y.bind(this._moveContent, this),
+                },
+            });
+        },
+
+        /**
+         * Move the content to the selected location
+         *
+         * @method _moveContent
+         * @protected
+         * @param {EventFacade} e
+         */
+        _moveContent: function (e) {
+            var app = this.get('app'),
+                initialActiveView = app.get('activeView'),
+                parentLocationId = e.selection.location.get('id'),
+                locationId = this.get('location').get('id'),
+                that = this,
+                notificationSuffixe = parentLocationId + locationId;
```

Comment:
instead of preparing a suffix (without a *e*), why don't you store here the identifier of the notification you'll use below ?